doc/cla/corporate/adhoc.md

```diff
@@ -6,17 +6,14 @@ I declare that I am authorized and able to make this agreement and sign this dec
 
 Signed,
 
-Damián Soriano ds@adhoc.com.ar https://github.com/damiansoriano
```

Comment:
please don’t remove members that have already made contributions, instead you can move then at the bottom of the file and add a note (like "up to …")